recipes/multilingual/vits_tts/train_vits_tts.py

```diff
@@ -73,15 +75,16 @@
     max_audio_len=160000,
     output_path=output_path,
     datasets=dataset_config,
-    characters={
-        "pad": "_",
-        "eos": "&",
-        "bos": "*",
-        "characters": "!¡'(),-.:;¿?abcdefghijklmnopqrstuvwxyzµßàáâäåæçèéêëìíîïñòóôöùúûüąćęłńœśşźżƒабвгдежзийклмнопрстуфхцчшщъыьэюяёєіїґӧ «°±µ»$%&‘’‚“`”„",
-        "punctuations": "!¡'(),-.:;¿? ",
-        "phonemes": None,
-        "unique": True,
-    },
+    characters=CharactersConfig(
+        characters_class="TTS.tts.models.vits.VitsCharacters",
+        pad="<PAD>",
+        eos="<EOS>",
```

Comment:
@Edresson what is the impact of this change ? so far only the character used for "eos" was in the character list.
does this trigger a change in behavior ?

Reply:
The new Tokenizer already deals with this. Currently `<EOS>` is the default token for "eos".